target/linux/ramips/dts/mt7621_xiaoyu_xy-c5.dts

```diff
@@ -0,0 +1,107 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "xiaoyu,xy-c5", "mediatek,mt7621-soc";
+	model = "XiaoYu XY-C5";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "xy-c5:green:work";
+			gpios = <&gpio0 8 GPIO_ACTIVE_LOW>;
+		};
+
+		led_sys: sys {
+			label = "xy-c5:green:sys";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+		};
+
```

Comment:
Please do not just mark comments as resolved if there is no response or change.

Reply:
okay, I'm new here and just don't know how to do